src/lib/util/value.c

```diff
@@ -1481,14 +1488,18 @@ ssize_t fr_value_box_from_network(TALLOC_CTX *ctx,
 
 	switch (type) {
 	case FR_TYPE_STRING:
-		if (fr_value_box_bstrndup(ctx, dst, enumv, (char const *)src, len, tainted) < 0) {
+		FR_DBUFF_EXTEND_LOWAT_OR_RETURN(&work_dbuff, len);
```

Comment:
I'm pretty sure this isn't needed. I mean `fr_value_box_bstrndup_dbuff `should return a negative integer if there's not enough space, so it's a duplicate check?